src/app/fdctl/run/tiles/fd_gossip.c

```diff
@@ -132,10 +120,24 @@ struct fd_gossip_tile_ctx {
   ulong       duplicate_shred_out_wmark;
   ulong       duplicate_shred_out_chunk;
 
+  fd_wksp_t * voter_in_mem;
+  ulong       voter_in_chunk0;
+  ulong       voter_in_wmark;
```

Comment:
so you changed votes to go to voter and you're using replay mem for wen restart msgs now right? ultimately they're all gossip CRDS values right?